integration-tests/plugins/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = on => {
 
     getLoginUrl: auth.getLoginUrl,
 
-    stubLogin: () => Promise.all([auth.stubLogin({}), elite2api.stubUser(), elite2api.stubUserCaseloads()]),
+    stubLogin: user => Promise.all([auth.stubLogin(user), elite2api.stubUser(user), elite2api.stubUserCaseloads()]),
```

Comment:
could you default this to `{}` here and avoid having to pass in empty object everywhere?

Reply:
yes